### PR TITLE
193016/split members

### PR DESCRIPTION
**Pull request by mmaciekk**
[AB#193016](https://unicef.visualstudio.com/ICTD-HCT-MIS/_boards/board/t/HOPE%20Team/Stories/?workitem=193016)

### CODE REVIEW

File: frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx [hunk 1]
```diff
@@ -29,8 +29,9 @@ import { isPermissionDeniedError, renderSomethingOrDash } from '@utils/utils';
 import { UniversalActivityLogTable } from '../../tables/UniversalActivityLogTable';
 import { PaymentRecordHouseholdTable } from '../../tables/payments/PaymentRecordAndPaymentHouseholdTable';
 import { HouseholdCompositionTable } from '../../tables/population/HouseholdCompositionTable/HouseholdCompositionTable';
-import { HouseholdIndividualsTable } from '../../tables/population/HouseholdIndividualsTable/HouseholdIndividualsTable';
+import { HouseholdMembersTable } from '../../tables/population/HouseholdMembersTable.tsx/HouseholdMembersTable';
```
Comment (patryk-dabrowski) [inline frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx:32]: why `HouseholdMembersTable.tsx`?

File: frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx [hunk 2]
```diff
@@ -164,7 +165,13 @@ export const PopulationHouseholdDetailsPage = (): React.ReactElement => {
       <HouseholdCompositionTable household={household as HouseholdNode} />
       <Container>
         {household?.individuals?.edges?.length ? (
-          <HouseholdIndividualsTable
+          <HouseholdMembersTable
+            choicesData={choicesData}
+            household={household as HouseholdNode}
+          />
+        ) : null}
+        {household?.individuals?.edges?.length ? (
```
Comment (patryk-dabrowski) [inline frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx:173]: duplicated condition

File: frontend/src/containers/tables/population/CollectorsTable.tsx/CollectorsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
+import TableCell from '@mui/material/TableCell';
```
Comment (patryk-dabrowski) [inline frontend/src/containers/tables/population/CollectorsTable.tsx/CollectorsTable.tsx:1]: the directory has extension `.tsx`

File: frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx (hunk 1 above)
Comment (mmaciekk) [reply frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx:32]: fixed

File: frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx (hunk 2 above)
Comment (mmaciekk) [reply frontend/src/containers/pages/population/PopulationHouseholdDetailsPage.tsx:173]: fixed

File: frontend/src/containers/tables/population/CollectorsTable.tsx/CollectorsTable.tsx (hunk 3 above)
Comment (mmaciekk) [reply frontend/src/containers/tables/population/CollectorsTable.tsx/CollectorsTable.tsx:1]: fixed

**Review by patryk-dabrowski — APPROVED** [review]

**Review by pavlo-mk — APPROVED** [review]

**Review by patryk-dabrowski — APPROVED** [review]

**Pull request merged**